Jenkinsfile

```diff
@@ -39,14 +39,13 @@ node {
         sh "docker build --tag ${image}:latest --tag ${image}:${tag} ."
       }
       stage ('Test') {
-        withEnv(["COMPOSE_FILE=docker-compose.test.yml"]) {
-          sh "docker-compose up --detach"
-          try {
-              sh "docker-compose exec -T app ./manage.py migrate"
-              sh "docker-compose exec -T app ./manage.py test"
-          } finally {
-              sh "docker-compose down --volumes --remove-orphans"
-          }
+        sh "docker-compose up --detach"
```

Comment:
I suspect this might not work, since the `docker-compose.yml` file doesn't know anything about the `mdnwebdocs/developer-portal:<tag>` images? I'm wondering if it might be best to delete this line, and then...

Reply:
I was going under the assumption that it will grab the images it built, it just seems very inefficient if we just did a docker build previously and now we are running a `docker-compose up --build --detach`

Perhaps it should be a second pass optimization thing.